From 89ee61454916aefa2d97ae7ea8dcc3b5668c2340 Mon Sep 17 00:00:00 2001
From: KoffeinFlummi <koffeinflummi@protonmail.com>
Date: Mon, 1 Oct 2018 21:07:09 +0200
Subject: [PATCH 4/5] Revert "Declare framework-side hostapd (will be done by
 treble app)"

This reverts commit 3a68ec19bc5c30cbf471ab2bce7c8d212390ec51.
---
 board-base.mk          |  1 -
 framework_manifest.xml | 12 ------------
 sepolicy/treble_app.te |  5 -----
 3 files changed, 18 deletions(-)
 delete mode 100644 framework_manifest.xml
 delete mode 100644 sepolicy/treble_app.te

diff --git a/board-base.mk b/board-base.mk
index efe5e68..4274be1 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -1,4 +1,3 @@
 TARGET_SYSTEM_PROP := device/phh/treble/system.prop $(TARGET_SYSTEM_PROP)
 BOARD_BLUETOOTH_BDROID_BUILDCFG_INCLUDE_DIR := device/phh/treble/bluetooth
 TARGET_EXFAT_DRIVER := exfat
-DEVICE_FRAMEWORK_MANIFEST_FILE := device/phh/treble/framework_manifest.xml
diff --git a/framework_manifest.xml b/framework_manifest.xml
deleted file mode 100644
index d0f6d40..0000000
--- a/framework_manifest.xml
+++ /dev/null
@@ -1,12 +0,0 @@
-<manifest version="1.0" type="framework">
-    <hal format="hidl">
-        <name>android.hardware.wifi.hostapd</name>
-        <transport>hwbinder</transport>
-        <version>1.0</version>
-        <interface>
-            <name>IHostapd</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
-</manifest>
-
diff --git a/sepolicy/treble_app.te b/sepolicy/treble_app.te
deleted file mode 100644
index ae0271e..0000000
--- a/sepolicy/treble_app.te
+++ /dev/null
@@ -1,5 +0,0 @@
-allow system_app hal_wifi_hostapd_hwservice:hwservice_manager { add find };
-allow system_app hidl_base_hwservice:hwservice_manager { add };
-allow system_app wifi_data_file:dir create_dir_perms;
-allow system_app wifi_data_file:file create_file_perms;
-
-- 
2.19.0

